ISSUE-TRIAGE.md

```diff
@@ -0,0 +1,106 @@
+# Triaging of issue
+[The contents have been sourced from [docker/docker project](https://github.com/docker/docker/blob/master/project/ISSUE-TRIAGE.md)]
```

Comment:
Good point. Remove it the cross reference --- I just meant you started with what was there no need to actually reference it.